client/src/templates/Challenges/classic/editor.tsx

```diff
@@ -175,6 +180,29 @@ const initialData: DataType = {
 };
 
 const Editor = (props: PropTypes): JSX.Element => {
+  // For the audio element
+  const player = useRef<{
+    sampler: Tone.Sampler | undefined;
+    noteIndex: number;
+    shouldPlay: boolean;
+  }>({
+    // eslint-disable-next-line no-undefined
+    sampler: undefined,
```

Comment:
```suggestion
    sampler?: Tone.Sampler;
    noteIndex: number;
    shouldPlay: boolean;
  }>({
```
AFAIK that achieves the same goal.